### PR TITLE
LLC, update automation, enable update swagger/README*.md

### PR DESCRIPTION
**Pull request by weidongxu-microsoft**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](../../CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: eng/mgmt/automation/generate_data.py [hunk 1]
```diff
@@ -94,54 +97,114 @@ def generate(
...
+            if filename.lower().startswith('readme') and filename.lower().endswith('.md'):
+                readme_yaml_found = False
+                readme_path = os.path.join(swagger_dir, filename)
+                with open(readme_path, 'r', encoding='utf-8') as f_in:
+                    content = f_in.read()
+                if content:
+                    yaml_blocks = re.findall(r'```\s?(?:yaml|YAML)\n(.*?)```', content, re.DOTALL)
+                    for yaml_str in yaml_blocks:
+                        yaml_json = yaml.safe_load(yaml_str)
+                        if 'low-level-client' in yaml_json and yaml_json['low-level-client']:
```
Comment (weidongxu-microsoft) [inline eng/mgmt/automation/generate_data.py:177]: Find the 1st yaml block with `low-level-client` defined, and update this block.

**Comment by weidongxu-microsoft** [comment]
Tested locally with
```
python eng/mgmt/automation/generate_data.py --input-file=https://github.com/Azure/azure-rest-api-specs/blob/main/specification/purview/data-plane/Azure.Analytics.Purview.Scanning/preview/2018-12-01-preview/scanningService.json --service=purview --module=azure-analytics-purview-scanning --credential-types=tokencredential --credential-scopes=https://purview.azure.net//.default
```

for purview scanning (update input-file and credential-scopes).

File: eng/mgmt/automation/generate_data.py [hunk 2]
```diff
@@ -94,54 +97,116 @@ def generate(
...
+                            # yaml block found, update
+                            yaml_json['input-file'] = [input_file]
+                            if title:
```
Comment (haolingdong-msft) [inline eng/mgmt/automation/generate_data.py:180]: title, credential_types and credential_types are the only three types we support for update?

Comment (weidongxu-microsoft) [reply eng/mgmt/automation/generate_data.py:180]: And `input-file`. Yes, so far.

Change service or module will result in a different folder.

**Review by haolingdong-msft — APPROVED** [review]

**Pull request merged**